R/step-subset-slice.R

```diff
@@ -119,6 +131,12 @@ slice_min.dtplyr_step <- function(.data, order_by, ..., n, prop, with_ties = TRU
   )
 }
 
+#' @export
+slice_min.data.table <- function(.data, order_by, ..., n, prop, with_ties = TRUE) {
+  .data <- lazy_dt(.data)
+  slice_min(.data, !!enexpr(order_by), ..., n = n, prop = prop, with_ties = with_ties)
```

Comment:
Should this be `{{ order_by }}`?

Reply:
I originally did this since the internal `slice_min_max()` helper captured `order_by` using `enexpr()`. Now a simple switch to `{{ order_by }}` causes the function to not work for some reason. I'll figure out why and make the necessary changes.